### PR TITLE
fix: fix typo in GitHub button to authenticate

### LINKED ISSUE #1065: Bug: typo on start page
### Describe the bug

There is a missing character in the Label on the left side of the screen of the start page.

image

Related PR: #1064

### Steps to reproduce

Go to https://insights.opensauced.pizza/start

### Affected services

insights.opensauced.pizza

### Platforms

Desktop

### Browsers

Chrome

### Environment

Production

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by geromegrignon**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [X] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description

This PR fixes a typo currently displaying **Authenicate** on [start](https://insights.opensauced.pizza/start) page.

## Related Tickets & Documents

Fixes #1065 

## Mobile & Desktop Screenshots/Recordings

image

## Added tests?

- [ ] 👍 yes
- [X] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [X] 🙅 no documentation needed

### CODE REVIEW

File: pages/start.tsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ import ChooseTimezoneIcon from "img/icons/choose-timezone.svg";
 import ChooseTimezoneActiveIcon from "img/icons/choose-timezone-active.svg";
 import ChooseInterestsIcon from "img/icons/choose-interests.svg";
 import ChooseInterestsActiveIcon from "img/icons/choose-interests-active.svg";
-import HighlightIcon from "img/icons/highlight-icon.svg";
+import HighlightIcon from "img/icons/highlixfght-icon.svg";
```
Comment (0-vortex) [inline pages/start.tsx:11]: think this is a typo :D

**Review by brandonroberts — APPROVED** [review]

File: pages/start.tsx (hunk 1 above)
Comment (geromegrignon) [reply pages/start.tsx]: oh sorry, I edited from github.com directly and missed it :/
It's fixed

**Review by 0-vortex — APPROVED** [review]
LGTM! 👍

**Comment by brandonroberts** [comment]
Thanks @geromegrignon!

**Pull request merged**